woocommerce/woocommerce-gutenberg-products-block
# Update patterns with the new images

## What

Fixes #11780 

## Why

This PR improves the default state of some patterns.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Add a new post.
2. Add all the patterns in the section below and ensure they match the screenshot.


## Screenshots or screencast



### Intro
#### Featured Category Cover Image
<img width="1249" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/1194a8ed-e63b-444a-bd68-e5b819ef75bb">

#### Hero Product Split

<img width="1488" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/5dd33ebd-8997-43cf-8553-4175c56b1d3e">

#### Just Arrived full hero 

<img width="1259" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/7078a613-775d-4206-bea6-5ce011fcab26">


### Featured selling
#### Featured Category Triple 

<img width="1209" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/efb1ae63-d1b1-4ae5-affd-1110b3731ae2">

#### Hero product 3 Split
![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/ece257a0-45a9-45e0-ac79-22cba57fe60a)


#### Hero Product Chessboard

 ![locale local](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/a7def2bd-2ea0-4c02-a6a8-953689d5ba2c)

#### Banner

![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/7261e4c4-2166-411f-96ac-0678500a4323)

#### Featured Category Focus
![image](https://github.com/woocommerce/woocommerce-blocks/assets/4463174/1070bab8-031e-4ab0-a0ec-1d6af2def513)



### About
#### Alterning image and text 
<img width="522" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/481a5cdf-5fd2-4a98-a077-82d0f42e38c8">

 
### Reviews
#### Testimonials single
<img width="1230" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/26bcb509-8c5c-470f-8d63-d26a20b3bb35">

### Social Media
#### Social Media

<img width="1164" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/4463174/bb98d7dc-64b4-463e-ae61-bdb5b3123c2a">





## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
